tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_deposit_request.py

```diff
@@ -17,6 +17,21 @@ def test_process_deposit_request_min_activation(spec, state):
     yield from run_deposit_request_processing(spec, state, deposit_request, validator_index)
 
 
+@with_electra_and_later
+@spec_state_test
+def test_process_deposit_request_extra_gwei(spec, state):
+    """The deposit amount must be at least 1 ETH and must be a multiple of gwei."""
+    validator_index = len(state.validators)
+    # An amount with some gwei (the +1 at the end)
+    amount = spec.EFFECTIVE_BALANCE_INCREMENT + 1
```

Comment:
Should this be instead? 
`    amount = spec.EFFECTIVE_BALANCE_INCREMENT + spec.Gwei(1)`